recipes/libcurl/all/conanfile.py

```diff
@@ -337,38 +396,39 @@ def _build_with_cmake(self):
         cmake.build()
 
     def package(self):
-        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        with tools.chdir(self.source_folder):
+            self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
 
-        # Execute install
-        if self.settings.compiler != "Visual Studio":
-            env_run = RunEnvironment(self)
+            # Execute install
+            if self.settings.compiler != "Visual Studio":
+                env_run = RunEnvironment(self)
```

Comment:
remove it. AutotoolsBuildEnvironment already uses RunEnvironment

Reply:
sure, but this was already here, I just indented the lines ..... 
are you  sure I can remove it?

Reply:
I think the original author did this for a reason, maybe because of when you run `conan package` you want to have, and need, the environment.
So I better leave it as it was